portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreation.testcase

```diff
@@ -1568,6 +1568,78 @@ definition {
 			value1 = "DYNAMIC SEGMENT");
 	}
 
+	@description = "Bug: LRAC-10972 | Automation ID: LRAC-11506 | Test Summary: Check Autocomplete Suggestions For All Criteria Types"
+	@priority = "3"
+	test CheckAutocompleteSuggestionsForAllCriteriaTypes {
+		task ("Sign in as new user") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+		}
+
+		task ("Go to AC Page DXP") {
+			ACUtils.navigateToSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
+		}
+
+		task ("Get URL") {
+			var myURL = Navigator.getCurrentURL();
+		}
+
+		task ("Switch to new property in AC") {
+			ACUtils.closeAllSessionsAndWait();
+
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = "${propertyName}");
```

Comment:
Hey @joshitagaki, so to access the property name you can use _${assignedPropertyName}_ because the _ACDXPSettings.connectDXPtoAnalyticsCloud_ macro returns a global value for every test that uses the macro.